cocos/tween/actions/action-manager.ts

```diff
@@ -457,7 +458,14 @@ export class ActionManager {
             locCurrTarget = this._currentTarget;
 
             const target = locCurrTarget.target;
-            if (target instanceof CCObject && !target.isValid) {
+            let isCCObject = target instanceof CCObject;
```

Comment:
It might be better to encapsulate a method to do this

Reply:
Yep, it is better. But to implement it in which file?

Reply:
Encapsulated it into Object.ts as a helper funcniton